core/src/main/java/org/mule/DefaultMuleEvent.java

```diff
@@ -83,7 +86,7 @@
     /** Mutable MuleEvent state **/
     private boolean stopFurtherProcessing = false;
     private int timeout = TIMEOUT_NOT_SET_VALUE;
-    private transient ResponseOutputStream outputStream;
```

Comment:
I'm not sure we can/should deprecate ResponseOutputStream, I haven't analysed where it is used.  But we can use OutputStream everywhere in MuleEvent I think, I don't see any issue with that. You?